python/lsst/ctrl/stats/data/dbEntry.py

```diff
@@ -33,15 +32,33 @@ def __init__(self, dbList):
         """
         # the DAG node name
```

Comment:
Some time ago I was told I needed to comment these, so I'm going to leave them as is for now.